test/new_relic/agent/llm/llm_event_test.rb

```diff
@@ -46,6 +46,17 @@ def test_event_attributes_returns_a_hash_of_assigned_attributes_and_values
       assert_equal('gpt-4', result['response.model'])
     end
 
+    def test_event_attributes_adds_custom_attributes
+      event = NewRelic::Agent::Llm::LlmEvent.new(id: 123)
+      event.vendor = 'OpenAI'
+      event.response_model = 'gpt-4'
+      event.metadata = {'Marathon' => '26.2', 'Ultra Marathon' => 'Ouch'}
+      result = event.event_attributes
+
+      assert_equal('26.2', result['Marathon'])
+      assert_equal('Ouch', result['Ultra Marathon'])
```

Comment:
I'm somewhat interested in adding an assertion to this test that makes sure the other attributes are also correctly assigned. Maybe something like:
```suggestion
      assert_equal('Ouch', result['Ultra Marathon'])
      assert_equal('OpenAI', result[:vendor])
```

I'm not sure if other tests would fail if we accidentally refactored the code in a way that overrode the attributes hash with just the metadata hash.

Reply:
I like it! Taking this commit suggestion